### ISSUE TITLE
Can't save a image to disk (Android)

### ISSUE THREAD

**Issue by Shin-NiL** [godotengine/godot#4648]
**Operating system or device - Godot version:**
Android 6.0.1, Godot 2.0.2 stable.

**Issue description** (what happened, and what was expected):
Trying to save a image on disk from Sprite returns the following error: 
```
05-13 06:03:54.089 14508 14521 I godot   : ERROR: save_image: Method/Function Failed, returning: ERR_CANT_OPEN
05-13 06:03:54.089 14508 14521 I godot   :    At: drivers/png/resource_saver_png.cpp:132.
```
The same project works just fine on PC.

**Steps to reproduce:**
add a Sprite node and set a texture via editor, then try to save the image to disk using the code:

```
var sprite = get_node("Sprite")
var texture = sprite.get_texture()
var image = texture.get_data()
image.save_png("user://tmp.png")
```

it's not likely permission related, because if I create a image via code, it works as expected:
```
var image2 = Image(64, 64, false, Image.FORMAT_RGBA)
image2.save_png("user://tmp2.png")
```

**Link to minimal example project** (optional but very welcome):
https://www.dropbox.com/s/oisqr3etx4wiioj/godot-share.zip

**Comment by vnen** [comment]
It's possible that the texture can't be retrieved from the GPU memory, so you there's nothing to save.

**Comment by Shin-NiL** [comment]
Is that an expected behavior on Android? Because it works fine on Windows and Linux.

**Comment by vnen** [comment]
Yes, that's a technical limitation of GLES2. See https://github.com/godotengine/godot/issues/2997#issuecomment-162258264.

**Comment by Shin-NiL** [comment]
Thanks @vnen, now it makes sense.

**Comment by vnen** [comment]
So I guess this have the same outcome of #2997: it's not possible to solve it. Things will likely get better after 3.0 with the new renderer.

**Comment by Zarasan** [comment]
Any update if this is fixed, I can't take screenshot in Godot 3.2.2 beta 2 on Android 10 either with GLES3 and GLES2

**Comment by Shin-NiL** [comment]
Yes, it was. I did not use 3.2.2 beta though.
